src/NetSparkle/Interfaces/FilterResult.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace NetSparkleUpdater.Interfaces
```

Comment:
This is a struct in the `Interfaces` folder and should move somewhere else. It can go into the `AppCastHandlers` folder (along with a namespace change), I think.